### PR TITLE
Restore gene panel functionality in oncoprint

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/ResultsViewPageStoreUtils.ts [hunk 1]
```diff
@@ -85,6 +89,49 @@ export function getOncoKbOncogenic(response:IndicatorQueryResp):string {
...
+export function computeGenePanelInformation(
+    genePanelData:GenePanelData[],
+    samples: Sample[],
+    patients: Patient[],
+    genes:Gene[]
+):GenePanelInformation {
+    const entrezToGene = _.keyBy(genes, gene=>gene.entrezGeneId);
+    const sampleInfo:GenePanelInformation["samples"] = {};
+    const patientInfo:GenePanelInformation["patients"] = {};
+    for (const sample of samples) {
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStoreUtils.ts]: not too important but i think use of reduce to build these collections would make this code clearer.  the reason is that you get simple assignment.   const sampleInfo = _.reduce(...).

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStoreUtils.ts]: good idea

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStoreUtils.ts]: done

**Comment by alisman** [comment]
@adamabeshouse please rebase and merge

**Pull request merged**